vuepress/docs/v6.3.2/tutorials/customize-the-platform/pda-tutorial.md

```diff
@@ -1,344 +1,233 @@
-plugin. The third provides a detailed technical overview for developers
-to add new functionality to a PDA implementation.
+# Entando Process Driven Applications Plugin
 
-# Terminology
+## Introduction
 
-## Micro Frontend
+The Entando Process Driven Applications (PDA) plugin is engineered to provide a rich and full-featured user experience while facilitating the management and completion of business processes and automation.  The solution template includes: 
 
-A Micro frontend architecture is an approach to developing web
-application as a composition of small frontend apps. Instead of writing
-a large monolith frontend application, the application is broken down
-into domain specific micro frontends, which are self-contained and can
-be developed and deployed independently.
+- A general purpose UX layer created from micro frontends and can be implemented for any business process or task engine. The UX layer can serve data via the included Entando integration adapter or by implementing a set of interfaces on the server side.
+- A Spring Boot microservice backend providing a pluggable interface for the injection of underlying processes or automation toolkits. The interfaces and steps for creating a new PDA backend implementation are described in further detail [here](link).
 
-## Process Automation Manager (PAM)
+This tutorial will guide you through installing the PDA plugin using the Entando Component Repository (ECR) and an Entando bundle. The goal of this exercise is to demonstrate the scope of process automation functionality enabled by integrating the following components: 
 
-A business process automation engine built and maintained by Red Hat
+- custom Micro Frontends
+- Process Automation Manager
+- Backend for Frontend
 
-## Business Process Modeling Notation (bpmn)
+Key elements of the template are reviewed in the [Application Details section](#application-details) below.
 
-Business Process Model and Notation (BPMN) is a graphical representation
-for specifying business processes in a business process model.
+## Installation
 
-## Decision Model and Notation (dmn)
+There are numerous assets installed as part of the Entando PDA plugin. Entando Bundles can include more or less components, depending on objectives. It is recommended that organizations develop guidelines for bundle sizing that fit the goals of their applications and teams.
 
-Decision Model and Notation is a standard published by the Object
-Management Group. It is a standard approach for describing and modeling
-repeatable decisions within organizations to ensure that decision models
-are interchangeable across organizations.
+### Prerequisites
 
-## Back End for Front End (BFF)
+- An Entando Application on any Kubernetes provider. Follow one of the [tutorials](../#operations) appropriate to your environment to install the Entando platform.
+- The ent command line tool, installed and connected to your Kubernetes instance.
```

Comment:
Add Red Hat PAM here since as-is they'll need their own PAM to start making use of the template.  It's definitely a prerequisite, just like JIRA Service Management for the Customer Portal.